lib/hex_web/web/templates/package.html.eex

```diff
@@ -50,6 +50,11 @@ build_tools = @current_release && @current_release.meta["build_tools"] || []
           </ul>
         </dd>
       <% end %>
+
+      <%= if @hexdocs_url do %>
+        <dt>Hexdocs</dt>
```

Comment:
Sorry for the back and forth, I talked to José and we found a design that we like. Lets go with your original approach, but with a slight difference: "Docs: online (download)".

"online" will have the hexdocs link. "download" will link to the tarballed documentation on S3. An example URL would be https://s3.amazonaws.com/s3.hex.pm/docs/decimal.tar.gz. You can build it with the `s3_url` and `s3_bucket` configuration values. The link will be broken at first because I need to setup redirects from decimal.tar.gz to decimal-LATEST.tar.gz.

https://github.com/hexpm/hex_web/blob/master/config/config.exs#L17-L18